cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/ConfigMapOperator.java

```diff
@@ -29,4 +32,22 @@ public ConfigMapOperator(Vertx vertx, KubernetesClient client) {
     protected MixedOperation<ConfigMap, ConfigMapList, DoneableConfigMap, Resource<ConfigMap, DoneableConfigMap>> operation() {
         return client.configMaps();
     }
+
+    @Override
+    protected Future<ReconcileResult<ConfigMap>> internalPatch(String namespace, String name, ConfigMap current, ConfigMap desired) {
+        try {
+            TreeMap a = new TreeMap(current.getData());
+            TreeMap b = new TreeMap(desired.getData());
+            if (a.toString().equals(b.toString())) {
+                log.debug("{} {} in namespace {} has not been patched because resources are equal", resourceKind, name, namespace);
+                return Future.succeededFuture(ReconcileResult.noop());
+            }
+            log.debug("{} {} in namespace {} has been patched", resourceKind, name, namespace);
```

Comment:
You can just call `super.internalPatch`. No need to reimplement the same code here.